pipelines/tekton/aiedge-e2e/aiedge-e2e.tensorflow-housing.pipelinerun.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  labels:
+    tekton.dev/pipeline: aiedge-e2e
+  generateName: aiedge-e2e-
+spec:
+  params:
+  - name: model-name
+    value: tensorflow-housing
+  - name: model-version
+    value: "1"
+  - name: s3-bucket-name
+    value: rhoai-edge-models
+  - name: gitServer
+    value: https://github.com
+  - name: gitOrgName
+    value: opendatahub-io
+  - name: gitRepoName
+    value: ai-edge
+  - name: containerfileRelativePath
+    value: pipelines/containerfiles/Containerfile.openvino.mlserver.mlflow
+  - name: fetch-model
+    value: "git"
```

```suggestion
    value: "s3"
```
Or overwrite this in the pipeline tests.
Currently we test S3 using both examples as this is more important than Git use case AFAIK. There is a plan to include also Git testing, of course, but just for now I would stick with the git.

---

@MarianMacik what's the value of testing S3 in both examples? Wouldn't the fact that it's used in one cover that pipeline task, and using git in the other one be additional coverage for the git clone task? My assumption was that they were of equal/similar importance.